packages/medusa/src/services/batch-job.ts

```diff
@@ -50,6 +54,30 @@ class BatchJobService extends BaseService<BatchJobService> {
     return cloned
   }
 
+  async create(data: BatchJobCreateProps): Promise<BatchJob> {
+    return await this.atomicPhase_(async (manager) => {
+      const batchJobRepo: BatchJobRepository = manager.getCustomRepository(
+        this.batchJobRepository_
+      )
+
```

Comment:
suggestion: Add TODO that we need to validate the data using BatchJobStrategy when implemented